data/projects/integrate-pyflakes-AST.md

```diff
@@ -0,0 +1,16 @@
+Integrate the [`pyflakes`](https://pypi.python.org/pypi/pyflakes)-enhanced [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree) into coala so that bears can access it, and create two bears that wrap the flake8 AST plugins [`flake8-future-import`](https://github.com/xZise/flake8-future-import) and [`flake8-string-format`](https://github.com/xZise/flake8-string-format) using the pyflakes-enhanced AST.
+
+#### Milestones
+
+##### GSOC 2017 CODING STARTS
+
+* The participate knows how to integrate pyflakes-AST into coala
+* The participant knows how to write coala-bears
+
+##### GSOC 2017 MIDTERM
+
+* Integrate pyflakes-enhanced AST into coala
```

Comment:
I think this pyflakes enhanced AST should be a "meta-bear", like `AnnotationBear` and `CLangBear`, i.e. `PyflakesASTBear`, and the wrapped flake8 extensions should then depend on `PyflakesASTBear`.